src/util/flssUtils.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable import/prefer-default-export */
+import flssLoanQuery from '@/graphql/query/flssLoansQuery.graphql';
+import createApolloClient from '@/api/apollo';
+
+export function fetchData(loanQueryFilters) {
+	const zeroLoans = false;
+	const newLoans = [];
+	const totalCount = 0;
+
+	this.apollo.query({
+		query: flssLoanQuery,
```

Comment:
@ryan-ludwig I'm not sure why I'm getting an error of `"TypeError: Cannot read property 'apollo' of undefined"`
I thought apollo was provided to all the components from the `main.js` file? 

The other things I've tried was creating an apollo client within the function. But i also got an error there b/c I hadn't configured the apolloClient() correctly. The other thing I tried was writing the utils file like:
```js
export default {
inject: ['apollo'],
methods: { 
fetchData()
}
}
```
so I wasn't running into the apollo error but  lenderFilterAlpha.vue threw an error that flss.Utils.default.fetchData() was not a function.

Reply:
also @emuvente or @dymonda21  would appreciate any eyes on how to get the function imported 🙏

Reply:
Instead of using `this.apollo` I would recommend have apollo be a second argument to the fetchData function:
```
export function fetchData(loanQueryFilters, apollo) {
	...
	apollo.query ({ ...
```
and when you're calling it from a component you can pass is the instance of apollo that the component has:
```
fetchData(filters, this.apollo);
```

The way you have the import defined looks correct, but you could also try it this way:
```
export default function fetchData(loanQueryFilters, apollo) { ... }

import fetchData from '@/util/flssUtils';
```

Reply:
thanks y'alls!